src/cmd/flux-job.c

```diff
@@ -1595,6 +1595,7 @@ struct attach_ctx {
     int eventlog_watch_count;
```

Comment:
Commit message for "flux-job: attach: always exit nonzero on fatal job exception"

s/if all job finishes/if a job finishes/